apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -382,6 +427,20 @@ export default function teacherSections(state=initialState, action) {
     };
   }
 
+  if (action.type === ASYNC_LOAD_BEGIN) {
+    return {
+      ...state,
+      asyncLoadComplete: false,
+    };
+  }
+
+  if (action.type === ASYNC_LOAD_SUCCESS || action.type === ASYNC_LOAD_FAILURE) {
```

Comment:
Hmm - this is definitely something I should revisit, I made them different actions so we could handle errors nicely.